source/HiringPanel.cpp

```diff
@@ -110,11 +110,13 @@ bool HiringPanel::KeyDown(SDL_Keycode key, Uint16 mod, const Command &command, b
 	if(key == 'h' || key == '=' || key == SDLK_RETURN || key == SDLK_SPACE)
 	{
 		player.Flagship()->AddCrew(min(maxHire, Modifier()));
+		player.Flagship()->SetDesiredCrew(player.Flagship()->Crew());
 		player.UpdateCargoCapacities();
 	}
 	else if(key == 'f' || key == '-' || key == SDLK_BACKSPACE || key == SDLK_DELETE)
 	{
 		player.Flagship()->AddCrew(-min(maxFire, Modifier()));
+		player.Flagship()->SetDesiredCrew(player.Flagship()->Crew());
```

Comment:
If the player reduces the `DesiredCrew` to the `Minimum Required Crew`, then let's set the `DesiredCrew` value to 0 (since the player is then likely not interested in hiring more crew than needed).